src/components/annotations/annotation-utilities.ts

```diff
@@ -283,3 +288,58 @@ export function getSparrowCurve(
   };
   return curveData;
 }
+
+export const getRowElement = (canvasElement: HTMLDivElement, rowId?: string) => {
+  if (rowId === undefined) return undefined;
+
+  const rowSelector = `[data-row-id='${rowId}']`;
+  const rowElements = canvasElement.querySelectorAll(rowSelector);
```

Comment:
You can use `canvasElement.querySelectorAll<HTMLElement>(tileSelector);` to avoid the need to cast later.